src/io/node.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-console */
+import process from 'node:process'
+import chalk from 'chalk'
+import nodeReleases from 'node-releases/data/release-schedule/release-schedule.json'
+
+function getLatestNodeVersion() {
+  const versions = Object.keys(nodeReleases)
```

Comment:
It will be better to get the latest node through `semver`.

Reply:
Yes, taking the last element of the array is not elegant. Do you have a suggestion how to use semver to solve this?

Reply:
```
return versions.reduce((_prev, _curr) => {
  const prev = semver.coerce(_prev);
  const curr = semver.coerce(_curr);
  return semver.gt(curr, prev) ? curr : prev;
})
```
Just a suggestion.

Reply:
Nice. Please checkout the last commit :slightly_smiling_face: